staging/operator-registry/internal/declcfg/declcfg_to_model_test.go

```diff
@@ -1,8 +1,10 @@
 package declcfg
 
 import (
+	"encoding/json"
 	"testing"
 
+	"github.com/operator-framework/operator-registry/internal/property"
```

Comment:
nit: ordering

Reply:
I thought about fixing this here, but didn't want to cause conflicts with future cherry picks.